config/settings.yml

```diff
@@ -825,6 +825,7 @@
     - PowerOffVM_Task
     :VmPoweredOnEvent:
     - PowerOnVM_Task
+    - PowerOnMultiVm_Task
```

Comment:
Is ```VmPoweredOnEvent``` the final event for ```PowerOnMultiVm_Task```?

Reply:
Yes or DrsVmPoweredOnEvent depending on the cluster configuration

Reply:
Added to that event as well.